README.md

```diff
@@ -29,21 +29,22 @@ API design. Because of this, all of the codebase is rapidly changing.
 
 ## Community, discussion, contribution, and support
 
+Cluster API is developed in the open, and is constantly being improved by our users, contributors, and maintainers. It is because of you that we are able to automate cluster lifecycle management for the community.
+
+Join us!
+
 - Chat with us on the Kubernetes [Slack](http://slack.k8s.io/) in the [#cluster-api][#cluster-api slack] channel
 - Subscribe to the [SIG Cluster Lifecycle](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle) Google Group for access to documents and calendars
```

Comment:
Should we also add something to this line to indicate that this is also an appropriate way to ask questions and consume project news (such as releases)?

Reply:
yes, i can add that!

Reply:
i added a note here for this